microsoft-edge/devtools-guide-chromium/whats-new/2023/03/devtools-111.md

```diff
@@ -0,0 +1,155 @@
+---
+title: What's new in DevTools (Microsoft Edge 111)
+description: "Remote debugging of Microsoft Edge, PWAs, and WebView2 controls on Xbox and HoloLens devices. Device Mode toolbar has emulation controls for themes and vision deficiencies. Performance tool unminifies file and function names while profiling. Styles tab easier to use in Focus Mode. Copy Selector Stats table from Performance tool. In Focus Mode, assistive tech announces tool moved to Quick View. And more."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 03/15/2023
+---
+# What's New in DevTools (Microsoft Edge 111)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+
+<!-- ====================================================================== -->
+## Remote debugging of Microsoft Edge, PWAs, and WebView2 controls on Xbox and HoloLens devices
+
+<!-- Subtitle: Get the Remote Tools for Microsoft Edge app from the Microsoft Store to set up remote debugging today! -->
+
+The **Remote Tools for Microsoft Edge** app is now available on Xbox and HoloLens devices.  The **Remote Tools for Microsoft Edge** app is a plug-in for the **Windows Device Portal for Desktop** that enables remote debugging for tabs in Microsoft Edge, Progressive Web Apps (PWAs), and WebView2 controls.  By using remote debugging, you can run web content on a remote device and debug it right from your development machine.
+
+![The Remote Tools for Microsoft Edge app in the Microsoft Store](./devtools-111-images/remote-tools-in-the-microsoft-store.png)
+
+With remote debugging enabled, you don't need to open DevTools on either HoloLens or Xbox devices any more.  Instead, you can connect to the remote Xbox or HoloLens device using the Device portal, open the `edge://inspect` page, and then select the `target:` (tab, PWA, or WebView2 control) that you want to use with DevTools.
```

Comment:
I don't think we need to call out the word target so specifically. I'm fine with italics or just leaving it unhighlighted
```suggestion
With remote debugging enabled, you don't need to open DevTools on either HoloLens or Xbox devices any more.  Instead, you can connect to the remote Xbox or HoloLens device using the Device portal, open the `edge://inspect` page, and then select the *target* (tab, PWA, or WebView2 control) that you want to use with DevTools.
```